SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-dec.adoc

```diff
@@ -1,6 +1,1080 @@
+
+If *pm:AbstractMetricDescriptor /@MetricAvailability* is set to "*Intr*" and if there are no continuously measured metrics to be exported, the field is set to the oldest *@DeterminationTime* of the episodic metrics to be exported in the same HL7 message.
+
+Note that the HL7 date and time format differs from the XML Schema date/time formats and requires a mapping accordingly (see also <<ref_expl_dt_mapping>>).
+
+|===
+
+====== OBR-8 Observation End Date/Time
+
+.R8008
+[sdpi_requirement#r8008,sdpi_req_level=shall]
+****
+A <<actor_somds_dec_gateway>> shall not use the OBR-8 field.
+****
+
+[#ref_dec_obr10]
+====== OBR-10 Collector Identifier
+
+.R8009
+[sdpi_requirement#r8009,sdpi_req_level=shall]
+****
+A <<actor_somds_dec_gateway>> shall set the OBR-10 field to the operator (user) information if available.
+
+The field shall be left empty if there is no valid SDC operator context.
+
+.Notes
+[%collapsible]
+====
+NOTE: <<ref_tbl_OBR10_mapping>> defines the mapping of the SDC operator context information to the data fields of the HL7 data type *XCN* used in the OBR-10 field.
+
+NOTE: The SDC operator context is only valid when the *pm:OperatorContextState/@ContextAssociation* is set to *"Assoc"* and a *pm:OperatorContextState/@BindingStartTime* is set.
+====
+****
+
+[#ref_tbl_OBR10_mapping]
+.OBR-10 Operator Information Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|OBR-10/XCN-1
+|ID Number
+|pm:OperatorContextState
+/pm:Identification
+/@Extension
+|The @Extension attribute contains the unique operator identifier.
+
+*Note that the field may contain a null value indicating that the identifier is missing.*
+
+|OBR-10/XCN-2
+|Family Name
+|pm:OperatorContextState
+/pm:OperatorDetails
+|HL7 data type *FN*
+
+|OBR-10/XCN-2.1
+|Surname
+|/pm:Familyname
+|
+
+|OBR-10/XCN-3
+|Given Name
+|pm:OperatorContextState
+/pm:OperatorDetails
+/pm:Givenname
+|
+
+|OBR-10/XCN-4
+|Second and Further Given Names or Initials
+|pm:OperatorContextState
+/pm:OperatorDetails
+/pm:Middlename
+|
+
+|OBR-10/XCN-6
+|Prefix (e.g. DR)
+|pm:OperatorContextState
+/pm:OperatorDetails
+/pm:Title
+|
+
+|OBR-10/XCN-9
+|Assigning Authority
+|pm:OperatorContextState
+/pm:Identification
+| HL7 data type *HD*
+
+|OBR-10/XCN-9.1
+|Namespace ID
+|/@Root
+|The @Root contains the unique identification of the HDO.
+
+*Note that if the HDO identifier is not defined, the XCN-9 field is left empty.*
+
+|===
+
+===== OBX - Observation/Result Segment
+The HL7 Observation/Result (OBX) segment requires a mapping from the SDC containment tree and metric items to the OBX segment fields.
+More information about the containment tree mapping can be found in *Appendix A Mapping ISO/IEEE 11073 Domain Information Model to HL7* in <<ihe_pcd_tf_2_2019>>.
+
```

Comment:
Add Reference to IHE PCD Tf-2 for OBX-1 like for the Height and Weight mapping

Reply:
Added OBX-1 section and reference as proposed.